packages/mdc-slider/index.js

```diff
@@ -116,6 +119,28 @@ export class MDCSlider extends MDCComponent {
       setTrackStyleProperty: (propertyName, value) => {
         this.track_.style.setProperty(propertyName, value);
       },
+      setMarkerValue: (value) => {
+        this.pinValueMarker_.innerText = value;
+      },
+      appendTrackMarkers: (numMarkers) => {
+        const frag = document.createDocumentFragment();
+        for (let i = 0; i < numMarkers; i++) {
+          const marker = document.createElement('div');
+          marker.classList.add('mdc-slider__track-marker');
+          frag.appendChild(marker);
+        }
+        this.trackMarkerContainer_.appendChild(frag);
+      },
+      removeTrackMarkers: () => {
+        while (this.trackMarkerContainer_.firstChild) {
+          this.trackMarkerContainer_.removeChild(this.trackMarkerContainer_.firstChild);
+        }
+      },
+      setLastTrackMarkersStyleProperty: (propertyName, value) => {
```

Comment:
I did quite get the question, I think `setLastTrackMarkersStyleProperty` is already an adapter method, do you mean you want it to be more generic so that it is not specific to last track marker?